src/mysql-util/src/tunnel.rs

```diff
@@ -139,7 +147,13 @@ impl Config {
             TunnelConfig::Ssh { config } => {
                 let (host, port) = self.address();
                 let tunnel = ssh_tunnel_manager
-                    .connect(config.clone(), host, port, self.ssh_timeout_config)
+                    .connect(
+                        config.clone(),
+                        host,
+                        port,
+                        self.ssh_timeout_config,
+                        self.in_task,
+                    )
```

Comment:
Uncovered. @nrainer-materialize I guess we should test mysql + ssh tunnel, and probably the same is missing for postgres. Do you want to take that (or have it planned already) or should I?

Reply:
Yes! As well as aws connections with `COPY TO S3` now!